generator/generator.go

```diff
@@ -96,6 +96,7 @@ var (
 	// Flags used for PathStruct generation only.
 	schemaStructPath        = flag.String("schema_struct_path", "", "The Go import path for the schema structs package. This should be specified if and only if schema structs are not being generated at the same time as path structs.")
 	generateWildcardPaths   = flag.Bool("generate_wildcard_paths", true, "Whether to generate methods for constructing wildcard paths.")
+	simplifyWildcardPaths   = flag.Bool("simplify_wildcard_paths", false, "Whether to omit the keys in the generated paths if all keys are wildcarded for a list node.")
```

Comment:
maybe "if all keys for a list node are wildcards"
avoids using "wildcard" as a verb

Reply:
If you change here, also change similarly below.

Reply:
Done.